api/base/serializers.py

```diff
@@ -44,6 +44,48 @@ def format_relationship_links(related_link=None, self_link=None, rel_meta=None,
     return ret
 
 
+class HideIfRetraction(ser.Field):
+    """
+    If node is retracted, this field will return None.
+    """
+    def __init__(self, field, **kwargs):
+        super(HideIfRetraction, self).__init__(**kwargs)
+        self.field = field
+        self.source = field.source
```

Comment:
In any case, it would be good to document proper usage to avoid the ambiguity above.